src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -0,0 +1,227 @@
+ * email submission: form, loading, confirmation, and error.
+ */
+export const NewsletterSignup = chakra(
+  forwardRef<any, NewsletterSignupProps>(
+    (
+      {
+        className,
+        confirmationText = "Thank you! You have successfully subscribed to our email updates! You can update your email subscription preferences at any time using the links at the bottom of the email.",
+        descriptionText = "Stay connected with the latest research news from NYPL, including information about our events, programs, exhibitions, and collections.",
+        formHelperText,
+        hiddenFields,
+        id,
+        isInvalidEmail = false,
+        newsletterSignupType = "whatsOn",
+        onSubmit,
+        title = "Sign Up for Our Newsletter!",
+        view = "form",
+        ...rest
+      },
+      ref?
+    ) => {
+      const viewType = view;
+      const [buttonClicked, setButtonClicked] = useState(false);
+      const [email, setEmail] = useState("");
+
+      // Hook into NYPL breakpoint
+      const { isLargerThanMobile } = useNYPLBreakpoints();
+      const focusRef = useRef<HTMLDivElement>(); // @todo Is this needed? It is a holdover from FeedbackBox
+      const styles = useStyleConfig("NewsletterSignup", {});
+      // The viewType may change as a result of something happening in the internalOnSubmit function.
+      const formView = viewType === "form";
+      const confirmationView = viewType === "confirmation";
+      const errorView = viewType === "error";
+      const iconColor = useColorModeValue(null, "dark.ui.typography.body");
+
+      const internalOnSubmit = (e) => {
+        e.preventDefault();
+        setButtonClicked(true);
+        const submittedValues = { email, ...hiddenFields };
+        onSubmit(submittedValues); // onSubmit comes from the consuming app
+      }; // Close internalOnSubmit
+
+      const privacyPolicy = (
+        <Link
+          href="https://www.nypl.org/help/about-nypl/legal-notices/privacy-policy"
+          type="external" // @todo The external link icon is slightly smaller in the Figma than the default served up by the Link component. I am unsure if/how to manipulate it.
+          id={"privacy"}
+        >
+          Privacy Policy
+        </Link>
+      );
+
+      return (
+        <Stack
+          direction={isLargerThanMobile ? "row" : "column"}
+          ref={ref}
+          __css={styles}
+          {...rest}
+        >
+          {/* Open info div */}
+          <Stack direction={isLargerThanMobile ? "row" : "column"} id={"info"}>
+            <Box
+              bg={getSectionColors(newsletterSignupType, "primary")}
+              id={"color-box"}
+              width={isLargerThanMobile ? ".75rem" : null} // @todo Not sure how to adapt this into the theme.ts and have it work the same.
+              height={isLargerThanMobile ? null : ".75rem"} // @todo ibid.
+            >
+              &nbsp;
```

Comment:
_I think_ you might be able to get the same result by having an empty Box with a wide right (or left) border. I'm not sure if having an empty `&nbsp;` is best practice for a visual cue.

Reply:
Resolved. I made changes to the style/html set up of the left/top of the component.